modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/item/selector/select_kb_articles.jsp

```diff
@@ -0,0 +1,306 @@
+									</span>
+								</liferay-ui:search-container-column-text>
+							</c:if>
+
+							<liferay-ui:search-container-column-text
+								cssClass="table-cell-expand-smallest table-cell-minw-100"
+								name="author"
+								value="<%= HtmlUtil.escape(PortalUtil.getUserName(curKBArticle)) %>"
+							/>
+
+							<liferay-ui:search-container-column-date
+								cssClass="table-cell-expand-smallest table-cell-ws-nowrap"
+								name="modified-date"
+								value="<%= curKBArticle.getModifiedDate() %>"
+							/>
+						</c:otherwise>
+					</c:choose>
+				</c:when>
+				<c:when test="<%= curKBFolder != null %>">
+
+					<%
+					PortletURL rowURL = PortletURLBuilder.create(
+						kbArticleItemSelectorViewDisplayContext.getPortletURL()
+					).setParameter(
+						"groupId", curKBFolder.getGroupId()
+					).setParameter(
+						"kbFolderId", curKBFolder.getKbFolderId()
+					).buildPortletURL();
+					%>
+
+					<c:choose>
+						<c:when test='<%= Objects.equals(kbArticleItemSelectorViewDisplayContext.getDisplayStyle(), "descriptive") %>'>
+							<liferay-ui:search-container-column-icon
+								icon="folder"
+								toggleRowChecker="<%= true %>"
+							/>
+
+							<liferay-ui:search-container-column-text
+								colspan="<%= 2 %>"
+							>
+
+								<%
+								Date createDate = curKBFolder.getCreateDate();
+
+								String createDateDescription = LanguageUtil.getTimeDescription(request, System.currentTimeMillis() - createDate.getTime(), true);
+								%>
+
+								<span class="text-default">
+									<liferay-ui:message arguments="<%= new String[] {HtmlUtil.escape(curKBFolder.getUserName()), createDateDescription} %>" key="x-modified-x-ago" />
+								</span>
+
+								<p class="font-weight-bold h5">
+									<a href="<%= rowURL %>">
+										<%= HtmlUtil.escape(curKBFolder.getName()) %>
+									</a>
+								</p>
+
+								<c:if test="<%= kbArticleItemSelectorViewDisplayContext.isSearchEverywhere() %>">
+									<h6 class="text-default">
+										<liferay-ui:message key="location" />:
+										<span class="text-secondary">
+											<clay:icon
+												symbol="<%= kbArticleItemSelectorViewDisplayContext.getGroupCssIcon(curKBFolder.getGroupId()) %>"
+											/>
+
+											<small><%= kbArticleItemSelectorViewDisplayContext.getGroupLabel(curKBFolder.getGroupId(), locale) %></small>
+										</span>
+									</h6>
+								</c:if>
+							</liferay-ui:search-container-column-text>
+						</c:when>
+						<c:when test='<%= Objects.equals(kbArticleItemSelectorViewDisplayContext.getDisplayStyle(), "icon") %>'>
+
+							<%
+							row.setCssClass("card-page-item card-page-item-directory " + row.getCssClass());
+							%>
+
+							<liferay-ui:search-container-column-text
+								colspan="<%= 2 %>"
+							>
+								<div class="card card-horizontal card-interactive card-interactive-secondary card-type-directory">
```

Comment:
Just wondering, instead of having this markup, couldn't we use <clay:horizontal-card> taglib? You have an example in https://github.com/ambrinchaudhary/liferay-portal/blame/master/modules/apps/item-selector/item-selector-web/src/main/resources/META-INF/resources/view_item_selector_view_descriptor.jsp#L73-L73